src/reader/widgets/ReaderWidget.vue

```diff
@@ -143,5 +149,24 @@
     & nav:last-child {
       margin-left: auto;
     }
+    ::v-deep .ball-pulse {
+      margin-left: auto;
+      padding-top: 40px;
+    }
+    ::v-deep .paginator {
```

Comment:
@paltman We discussed this in relation to DS a few weeks back, but we have an issue where we scroll down the reader and then paginate to the next passage.

When the new passage loads, we maintain the previous scroll position:

https://cl.ly/dde0870ee96a/Screen%20Recording%202020-06-02%20at%2004.48%20PM.gif

I think the tweak you made to DS would suffice here:
https://router.vuejs.org/guide/advanced/scroll-behavior.html

```javascript
// in router.js
scrollBehavior() {
    return { x: 0, y: 0 };
  },
```

Reply:
(This behavior exists on `master`, but I figure we could just sneak it in here since you're doing other pagination work)

Reply:
@paltman the scrollBehavior hook wasn't it.

What do you think about the approach taken in c743346?

Reply:
@jacobwegner that seems perfectly reasonable to me.  it's the way to do precisely what we want to happen.

Reply:
🎉